zulipterminal/core.py

```diff
@@ -109,16 +109,17 @@ def exit_popup(self) -> None:
         self.loop.widget = self.view
 
     def show_help(self) -> None:
-        help_view = HelpView(self)
-        self.show_pop_up(help_view, "Help Menu (up/down scrolls)")
+        title = 'Help Menu (up/down scrolls)'
+        help_view = HelpView(self, len(title))
+        self.show_pop_up(help_view, title)
 
     def show_msg_info(self, msg: Message) -> None:
-        msg_info_view = MsgInfoView(self, msg)
-        self.show_pop_up(msg_info_view,
-                         "Message Information (up/down scrolls)")
+        title = 'Message Information (up/down scrolls)'
+        msg_info_view = MsgInfoView(self, msg, len(title))
+        self.show_pop_up(msg_info_view, title)
 
     def show_stream_info(self, color: str, name: str, desc: str) -> None:
-        show_stream_view = StreamInfoView(self, color, name, desc)
+        show_stream_view = StreamInfoView(self, color, desc, len('# ' + name))
         self.show_pop_up(show_stream_view, "# {}".format(name))
```

Comment:
Is there a reason you didn't tidy this title duplication in addition to the other title changes?

Reply:
Thanks for pointing it out. I missed it.